Integrations/build.gradle

```diff
@@ -87,20 +87,20 @@ def runInDocker = { String name, String sourcePath, List<String> command, Closur
                 into 'python/configs'
             }
         }
-        parentContainers = [project(':docker-runtime-base').tasks.findByName('buildDocker')] // deephaven/runtime-base
+        parentContainers = [project(':docker-server').tasks.findByName('buildDocker-server')] // deephaven/server
 
         imageName = 'deephaven/py-integrations:local-build'
 
         addConfig(it)
 
         dockerfile {
             // set up the container, env vars - things that aren't likely to change
-            from 'deephaven/runtime-base:local-build'
+            from 'deephaven/server:local-build'
             runCommand '''set -eux; \\
                       pip3 install unittest-xml-reporting==3.0.4;\\
                       mkdir -p /out/report;\\
                       mkdir -p /workspace/cache/classes'''
-            environmentVariable 'DEEPHAVEN_CLASSPATH', '/classpath/*:/classpath'
+            environmentVariable 'DEEPHAVEN_CLASSPATH', '/classpath/*:/classpath:/opt/deephaven/server/lib/*:/opt/deephaven/server/lib/'
```

Comment:
This line looks wrong to me.  In my PR, I had to include a version number in the path.

Reply:
/opt/deephaven/server is a symlink to /opt/deephaven/server-{ver}, and should always be present, using it is safe and avoids having to handle versions at run time